.cargo/config

```diff
@@ -1,4 +1,4 @@
 [alias]
 wasm = "build --release --target wasm32-unknown-unknown"
-unit-test = "test --lib"
+unit-test = "test --lib --features=enable-test-utils"
```

Comment:
This is a better place to define how unit-test works than my previous strategy of doing it in the Makefile itself